src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -523,6 +826,8 @@ unique_ptr<CreateTableInfo> TableCatalogEntry::Deserialize(Deserializer &source)
 	info->schema = reader.ReadRequired<string>();
 	info->table = reader.ReadRequired<string>();
 	info->columns = reader.ReadRequiredSerializableList<ColumnDefinition, ColumnDefinition>();
+	info->generated_columns =
```

Comment:
Making this a required field breaks the storage tests, can this be added as an optional field at the very end of the FieldWriter?

Reply:
I'm not entirely sure what you mean by that
When I look at FieldWriter, there isn't an option to WriteOptionalSerializableList
Should I try an optional boolean instead to indicate whether this has generated columns?